config/locales/views.fr.yml

```
@@ -511,13 +511,18 @@ fr:
       title: solicitations traitées
     solicitation:
       cancel: Annuler
+      create_diagnosis: créer une analyse
```

```suggestion
      create_diagnosis: Démarrer une analyse
```
On dit “démarrer” à d’autres endroits dans l’interface, autant être cohérents